File: sdk/csharp/libraries/microsoft.bot.solutions/Authentication/AuthenticationResponses.cs
// https://docs.microsoft.com/en-us/visualstudio/modeling/t4-include-directive?view=vs-2017
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Microsoft.Bot.Solutions.Responses;

namespace Microsoft.Bot.Solutions.Authentication
{
    /// <summary>
    /// Contains bot responses.
    /// </summary>
    public class AuthenticationResponses : IResponseIdCollection
    {
        // Generated accessors
        public const string SkillAuthenticationTitle = "SkillAuthenticationTitle";
        public const string SkillAuthenticationPrompt = "SkillAuthenticationPrompt";
        public const string AuthProvidersPrompt = "AuthProvidersPrompt";
        public const string ConfiguredAuthProvidersPrompt = "ConfiguredAuthProvidersPrompt";
        public const string ErrorMessageAuthFailure = "ErrorMessageAuthFailure";
        public const string NoLinkedAccount = "NoLinkedAccount";
        public const string LoginButton = "LoginButton";
        public const string LoginPrompt = "LoginPrompt";
    }
}

File: sdk/typescript/libraries/bot-solutions/src/authentication/authenticationResponses.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { join } from 'path';
import { IResponseIdCollection } from '../responses';

export class AuthenticationResponses implements IResponseIdCollection {
    // Generated accessors
    public readonly name: string = AuthenticationResponses.name;
    public static readonly pathToResource: string = join(__dirname, 'resources');
    public static readonly skillAuthenticationTitle: string = 'SkillAuthenticationTitle';
    public static readonly skillAuthenticationPrompt: string = 'SkillAuthenticationPrompt';
    public static readonly authProvidersPrompt: string = 'AuthProvidersPrompt';
    public static readonly configuredAuthProvidersPrompt: string = 'ConfiguredAuthProvidersPrompt';
    public static readonly errorMessageAuthFailure: string = 'ErrorMessageAuthFailure';
    public static readonly noLinkedAccount: string = 'NoLinkedAccount';
    public static readonly loginButton: string = 'LoginButton';
    public static readonly loginPrompt: string = 'LoginPrompt';
}
